app/src/main/AndroidManifest.xml

```diff
@@ -22,13 +22,13 @@
         android:theme="@style/AppTheme">
 
         <activity
-            android:name=".ui.MainActivity"
+            android:name="com.xmartlabs.base.ui.MainActivity"
```

Comment:
Maybe leave them here for now, so when running the project it at least opens an empty view?